policy/modules/system/systemd.te

```diff
@@ -2042,6 +2070,12 @@ tunable_policy(`systemd_tmpfiles_manage_all',`
 ')
 
 optional_policy(`
+	colord_read_lib_files(systemd_tmpfiles_t)
+	colord_relabel_lib(systemd_tmpfiles_t)
+')
+
+optional_policy(`
+	dbus_manage_lib_files(systemd_tmpfiles_t)
 	dbus_read_lib_files(systemd_tmpfiles_t)
 	dbus_relabel_lib_dirs(systemd_tmpfiles_t)
 ')
```

Comment:
These should be moved to `systemd_tmpfilesd_managed()` calls on the types in the colord and dbus modules.